eox_tenant/admin.py

```diff
@@ -145,14 +148,29 @@ class RouteAdmin(admin.ModelAdmin):
     Route model admin.
     """
 
+    formfield_overrides = {
+        models.ForeignKey: {'widget': forms.TextInput},
+    }
+
     list_display = [
         "domain",
+        "config_link",
     ]
 
     search_fields = [
         "domain"
     ]
 
+    def config_link(self, route):
+        """
+        Helper method to display a link to the related config model.
+        """
+        # pylint: disable=protected-access
+        url = reverse('admin:%s_%s_change' % (route._meta.app_label, "tenantconfig"), args=[route.config.id])
+        return u'<a href="%s">%s</a>' % (url, route.config.__unicode__())
```

Comment:
Please, add a break line here, it's a little bit confusing. @diegomillan

Reply:
Done